frontend/src/components/jobs/result/JobInfoCard.jsx

```diff
@@ -39,12 +44,26 @@ export function JobInfoCard({ job }) {
             md={10}
           >
             <h3>
-              {job.is_sample ? (
-                <VscFile className="me-1" />
-              ) : (
-                <VscGlobe className="me-1" />
-              )}
-
+              {job.is_sample && <VscFile className="me-1" />}
+              {job?.observable_classification ===
+                ObservableClassifications.IP &&
+                country.countryCode && (
+                  <span className="px-1">
+                    {countryIcon}
+                    <UncontrolledTooltip
+                      placement="right"
+                      target={`Icon-${country.countryCode.toLowerCase()}`}
+                    >
+                      {country.countryName}
+                    </UncontrolledTooltip>
+                  </span>
+                )}
+              {!job.is_sample &&
+                (job?.observable_classification !==
+                  ObservableClassifications.IP ||
+                  (job?.observable_classification ===
+                    ObservableClassifications.IP &&
+                    !country.countryCode)) && <VscGlobe className="me-1" />}
```

Comment:
I think some of the counditions could be grouped: if is_sample else if is IP && coutry code else ...